src/components/FilterPanel/FilterPanelCheckboxWithOverflowMenu/_index.scss

```diff
@@ -0,0 +1,43 @@
+////
+/// Filter panel checkbox with overflow menu styles.
+/// @group filter-panel
+/// @copyright IBM Security 2020
+////
+
+@import '../../Component/mixins';
```

Comment:
Would you also mind including spacing and tokens imports here? This will silently fail because they may have been included elsewhere in the Sass import order.